salt/orch/update.sls

```diff
@@ -130,7 +130,7 @@ etcd-setup:
       - kube-apiserver.stop
       - kube-controller-manager.stop
       - kube-scheduler.stop
-      - docker.stop
+      - {{ salt.caasp_cri.cri_service_name() + '.stop' }}
```

Comment:
All these dynamic calculations of SLS names do not look right to me. Maybe we should move all the different CRI drivers under the same hierarchy, something similar to what we have for the `addons`:

```Shell

    /cri
        init.sls
        stop.sls
        /docker
            init.sls
            stop.sls
        /crio
            init.sls
            stop.sls
```

So we could just invoke `cri.stop` and `cri.stop` would invoke the right `stop` in `docker` or `crio`...